### PR TITLE
fix(VSlider): add label

### LINKED ISSUE #15732: [Bug Report][3.0.0-beta.10] v-slider's label doesn't show
### Environment
**Vuetify Version:** 3.0.0-beta.10
**Vue Version:** 3.2.38
**Browsers:** Firefox 103.0
**OS:** Windows 10

### Steps to reproduce
Go to vuetify 3 slider doc: https://next.vuetifyjs.com/en/components/sliders/#colors

### Expected Behavior
Labels should appear

### Actual Behavior
Labels don't show at all

### Reproduction Link
<a href="https://codepen.io/talw3g/pen/YzeWYEN" target="_blank">https://codepen.io/talw3g/pen/YzeWYEN</a>

### PR DESCRIPTION
**Pull request by nekosaur**
closes #15732

## Markup:

<details>

```vue
<template>
  <div class="pa-4">
    <v-slider label="Foo" />

    <v-slider label="Foo" prepend-icon="mdi-home" />

    <v-slider label="Foo">
      <template #label>
        LABEL
      </template>
    </v-slider>

    <v-slider label="Foo">
      <template #prepend>
        PREPEND
      </template>
    </v-slider>

    <v-range-slider label="Foo" />

    <v-range-slider label="Foo" prepend-icon="mdi-home" />
  </div>
</template>
```
</details>

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist:

- [ ] The PR title is no longer than 64 characters.
- [ ] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and backwards compatible changes and `next` for non-backwards compatible changes).
- [ ] My code follows the code style of this project.
- [ ] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)

### CODE REVIEW

File: packages/vuetify/src/components/VInput/VInput.tsx [hunk 1]
```diff
@@ -129,14 +129,14 @@ export const VInput = genericComponent<new <T>() => {
         >
           { hasPrepend && (
             <div key="prepend" class="v-input__prepend">
+              { slots.prepend?.(slotProps.value) }
```
Comment (KaelWD) [inline packages/vuetify/src/components/VInput/VInput.tsx:132]: Should we swap the order of append too for consistency?

File: packages/vuetify/src/components/VSlider/VSlider.sass [hunk 2]
```diff
@@ -35,8 +36,8 @@
 // Modifiers
 .v-slider.v-input--horizontal
   align-items: center
-  margin-left: $slider-horizontal-left
-  margin-right: $slider-horizontal-right
+  margin-inline-start: $slider-horizontal-left
+  margin-inline-end: $slider-horizontal-right
```
Comment (KaelWD) [inline packages/vuetify/src/components/VSlider/VSlider.sass:40]: rename these variables?

**Review by johnleider — APPROVED** [review]
lgtm

File: packages/vuetify/src/components/VSlider/VSlider.tsx [hunk 3]
```diff
@@ -90,6 +91,12 @@ export const VSlider = defineComponent({
         >
           {{
             ...slots,
+            prepend: props.label ? slotProps => (
+              <>
+                <VLabel class="v-slider__label">{ props.label }</VLabel>
```
Comment (KaelWD) [inline packages/vuetify/src/components/VSlider/VSlider.tsx:96]: Other components also have a label slot

File: packages/vuetify/src/components/VInput/VInput.tsx (hunk 1 above)
Comment (nekosaur) [reply packages/vuetify/src/components/VInput/VInput.tsx:132]: probably?

**Pull request merged**